it.js

```diff
@@ -1,821 +1,821 @@
+custom.map.upload.limit Fallito. È stato raggiunto il limite di 500k caratteri dei dati della mappa
+custom.map.missing Seleziona almeno una partita o mappa
+custom.mode.missing Seleziona almeno una modalità
+custom.class.missing Seleziona almeno una classe
+server.config.maxPlayers Giocatori max.
+server.config.minPlayers Giocatori min.
+server.config.lives Vite
+server.config.gameTime Minuti
+server.config.gravMlt Gravità
+server.config.tickRI Tickrate alto
+server.config.maxPS Server da 40 giocatori
+server.config.promServ Promuovi server
+server.config.jumpMlt Forza salto
+server.config.airStrf Strafe aereo
+server.config.deltaMlt Velocità tempo
+server.config.strafeSpd Velocità strafe
+server.config.fiRat Cadenza di tiro
+server.config.reSpd Velocità ricarica
+server.config.slidTime Tempo scivolata
+server.config.fixMov Movimento fisso
+server.config.slidSpd Velocità scivolata
+server.config.healthMlt Moltiplicatore vita
+server.config.impulseMlt Impulso armi
+server.config.nameTeam1 Nome squadra 1
+server.config.nameTeam2 Nome squadra 2
+server.config.nameTeam3 Nome squadra 3
+server.config.nameTeam4 Nome squadra 4
+server.config.nameTeam5 Nome squadra 5
+server.config.t1Dmg Danni squadra 1
+server.config.t2Dmg Danni squadra 2
+server.config.t3Dmg Danni squadra 3
+server.config.t4Dmg Danni squadra 4
+server.config.t5Dmg Danni squadra 5
+server.config.selTeam Scelta squadra
+server.config.allowSpect Spettatori
+server.config.killRewards Premi uccisioni
+server.config.headshotOnly Solo headshot
+server.config.noSecondary Disabilita secondarie
+server.config.noStreaks Disabilita serie
+server.config.canSlide Scivolare
+server.config.wallJP Forza salto dal muro
+server.config.autoJump Salto automatico
+server.config.bDrop Cadenza proiettili
+server.config.thirdPerson 3a persona
+server.config.requireCaptcha Richiedi Captcha
+server.config.nameTags Nascondi Nametags
+server.config.kCams Camera uccisione
+server.config.noReload Niente ricarica
+server.config.private Privata
+server.config.krRewards Premi KR
+server.config.forceDefault Forza impostazioni predefinite
+server.config.warmupTime Tempo riscaldamento
+server.config.gamRounds Rounds
+server.config.objtvTime Tempo obiettivo
+server.config.forceSpawn Tempo rinascita automatica
+server.config.intermTmr Pausa (s)
+server.config.scoreLimit Limite punteggio
+server.config.keepTScore Tieni punti squadra
+server.config.forceC Forza la classe
+server.config.logTim Tempo (Gara, Parkour)
+server.config.hitBoxPad Dimensione Hitbox
+server.config.hpRegen Rigenerazione vita
+server.config.disableB Disabilita bordi
+server.config.throwMel Armi da mischia lanciabili
+server.config.noCosm Disabilita cosmetici
+server.config.tstCmp Modalità test
+server.config.limitClasses Limita classi
+server.config.tmSize Dimensione squadra
+server.config.fallDmg Danno da caduta
+server.config.fallDmgThr Taglio del danno da caduta 
+server.config.noDraws Nessun disegno

```

Comment:
"Disegno" is incorrect, the "draw" is "pareggio". "Senza pareggi" or "Nessun pareggio" would be best.